### DISCUSSION TITLE
`Find File` doesn't work

### DISCUSSION THREAD

**Discussion by abcvav** [LazyVim/LazyVim#5205] [Q&A] [answered]
After entering `nvim` to start neovim, I press `f` to open "Find File", but no files can be searched. The dialog box prompts "Unable to stat file find".

Has anybody encountered this problem? Anyone knows how to solve this?

![image](https://github.com/user-attachments/assets/7eb81e86-2081-4dde-a9a2-35c5c90d7dc3)

**Comment by folke** [comment]
Read that warning message. It tells you to install fdfind or ripgrep.

**Reply by abcvav** [reply]
Should I install them manually, or is there a quick installation method in LazyVim? 😢

**Reply by dpetka2001** [reply, marked as answer]
Yes, you have to install them manually. These are packages for your OS after all, just like how you installed Neovim. 

You might want to have a look at [mise](https://github.com/jdx/mise), it's a tool for installing various apps. It doesn't have everything, but it definitely has `ripgrep` and `fd`.

Otherwise, you would have to read the docs of the package you want to prefer to see which available installation methods exist for your OS.

**Reply by abcvav** [reply]
After installing these apps, the problem is solved, thank you so much 😄

**Comment by caio-ishikawa** [comment]
Running into the same issue and I have both fdfind and ripgrep. Any ideas?

**Reply by ibhagwan** [reply]
Unless you’re running an old version of fzf-lua you can’t be running into the same issue, the POSIX `-printf` was removed, meaning the files picker works on MacOS even without installing fd/rg (using the default find):
https://github.com/ibhagwan/fzf-lua/blob/3942ec636a7538a9ddf7c9ebae4919fcb5342b75/lua/fzf-lua/defaults.lua#L352
